packages/apollo-server-core/src/__tests__/runHttpQuery.test.ts

```diff
@@ -29,6 +29,7 @@ describe('runHttpQuery', () => {
         query: '{ testString }',
       },
       options: {
+        debug: false,
```

Comment:
This was required to avoid returning a stack trace, which the test didn't expect. Without running through ApolloServer first, the `nodeEnv` stuff never runs, so it doesn't get setup correctly for this test. This avoids that.